prog/download_boot_image.rb

```diff
@@ -26,9 +26,8 @@ def blob_storage_client
   end
 
   label def wait_draining
-    unless vm_host.vms.empty?
-      nap 15
-    end
+    nap 15 unless vm_host.vms_dataset.where(boot_image: image_name).all.empty?
```

Comment:
For VMs and Postgres images, we don't use bdev_ubi. It is plain ext4 which means deleting the source file after we copy it should work fine. Maybe we should add those to the condition above, so that existence of them would be ignored in wait_draining as well.

Reply:
Currently, we use bdev_ubi for all boot storage volumes.